app-center-webapps/src/main/webapp/skin/less/app-center.less

```diff
@@ -16,16 +16,8 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 @import "./variables.less";
 
+@borderColor: var(--allPagesBorderColor, @greyColorDefault);
```

Comment:
This one is already defined in platform-ui. So It should not be redefined here.

Reply:
Okay will fix it